### DISCUSSION TITLE
Polish Language Leader

### DISCUSSION THREAD

**Discussion by Uriziel01** [OHF-Voice/intents#387] [Language Leaders]
Hi,
as I see the position seems to still be vacant?

I would like to apply for the position as Polish language leader. I’m a native Polish speaker and have over 14 years of experience as a software developer, PRs reviews are something I do almost every day.

Looking forward to contributing! 🥳

**Comment by synesthesiam** [comment]
Thanks, @Uriziel01! Adding you as language leader now.

**Comment by gm-krk** [comment]
Hello
If you need another person to work out Polish langauge I can contribute as a regular supporter. We have the leader alreday ! :) 
I us HA for a year now and have some CompScience background which could be benefitial.

**Comment by synesthesiam** [comment]
@gm-krk @Uriziel01 Would either of you be able to translate the new responses into Polish?

**Reply by gm-krk** [reply]
I can but need some intro on how to do this / where to put translations. 
Do not have experience with github ...

**Reply by synesthesiam** [reply]
Sure, the files to translate are here: https://github.com/home-assistant/intents/tree/main/responses/pl
Do you know how to create a fork?

**Comment by gm-krk** [comment]
Hi

not, really.

I can try to find out but wanted to make sure that I do not break/ misuse
the repository.

Do you have available a short guideline or we can talk later today (after
19) ?

Best regards, Grzegorz

śr., 25 sty 2023 o 00:10 Michael Hansen ***@***.***>
napisał(a):

**Comment by Uriziel01** [comment]
@gm-krk @synesthesiam Hi guys, sorry for delayed response, lot's on my head last few days. I can do the translations today's evening.

@gm-krk Do not worry, it's really hard to break anything on github when you lack permissions to do so, just try to find some resources (there are nice videos even in Polish) about general usage of Github's pull request and forking in general. If you want I can also send you some valuable resources?

**Comment by gm-krk** [comment]
Hi

Please send me resources you find best
I will go through it first.
@Paweł, Can you leave me at least one translation? I will use is as a
training :)

Thanks, Grzegorz



śr., 25 sty 2023 o 08:37 Paweł Borecki ***@***.***>
napisał(a):

**Comment by Uriziel01** [comment]
Sure, no problem Grzegorz. I'll do both in the evening.

**Comment by Hrupek** [comment]
Hi, if you need more people to help with Polish let me know. I am willing to help.

**Comment by rrakso** [comment]
@gm-krk I can give you a quick tutorial on how to add new translations here ;) We can connect via HA Discord or somehow hahaha. Just let me know if you still need some help ;)

**Comment by Uriziel01** [comment]
Hi, welcome back everyone. For the last few days I was sick in bed with fever so could not help you with anything. I'll be going through your PRs and comment throughout to day to catch up with everyting.

**Comment by gm-krk** [comment]
Hello

Yes, I would like to do some translations.
Can you point me how/ where to start?

Thanks, Grzegorz

pon., 30 sty 2023 o 16:46 Oskar Jaskólski ***@***.***>
napisał(a):

**Comment by synesthesiam** [comment]
@Uriziel01 when you get a chance, could you please look into translating the `HassGetState` sentences/responses?

**Reply by witold-gren** [reply]
Hey, I finally updated all files. The last one is `binary_sensor_HassGetState.yaml` - after my PR will be check and close I will start working with prepare this last update https://github.com/home-assistant/intents/pull/1212

**Comment by witold-gren** [comment]
Hey @synesthesiam, could you also add me to as a language leader on Polish translations? 😀 I think it will be much easier for me to add more of small changes instead of one big one. I think I'll take a look at it this week in `HassGetState` sentences/responses.

**Comment by Uriziel01** [comment]
Sorry @synesthesiam, I've missed the notification about your comment. Will do 🙂

**Comment by Uriziel01** [comment]
@nikito7 May I ask why you keep just accepting PRs for a language you are not a language leader for? I've got some comments for the latest PR from @witold-gren but it's a second time when you just accept a PR in my language without a single word of comment. Thank you for clarification.

**Reply by nikito7** [reply]
Ok sorry. Because this sometimes could be hard.
You can fix little things now. Sorry again.

**Comment by synesthesiam** [comment]
Added @witold-gren as a language leader :slightly_smiling_face: 
Once you've been invited to the Home Assistant organization, you can begin merging PR's.

**Comment by Uriziel01** [comment]
Thanks @synesthesiam, @witold-gren does some really great stuff in his PRs so this will definitely help a lot 😃
